getnacelle/nacelle-react
# chore: remove gatsby-source-nacelle

**Story:** [LS-1486](https://nacelle.atlassian.net/browse/LS-1486) 

### Type of Change


### What is being changed and why?

Removes `gatsby-source-nacelle`. This package is being moved to `getnacelle/nacelle-js` in PR: https://github.com/getnacelle/nacelle-js/pull/72
Updates `gatsby-source-nacelle` README to link to new location (pending PR above)

### How to Test

1. Run `npm run bootstrap`
2. Run `npm run test:ci` to ensure tests run with no errors.
3. Run `npm run build` to ensure packages build with no errors.
